etc/conf.d/server.lua

```diff
@@ -39,5 +41,18 @@ opts = {
   redirect_uri_scheme = "https",
   logout_path = "/logout",
   redirect_after_logout_uri = "https://sso.mozilla.com/forbidden",
+  -- The following options are used to verify a user session should be kept running and that the user is still valid
+  -- refresh_session_interval will 302 the user's browser every X amount of seconds (here, 900) transparently
+  -- renew_access_token_on_expiry will use an access or refresh token with a server-side request. If you use the later
+  -- make sure you enable all 3 options: renew_access_token_on_expiry,access_token_expires_in,access_token_expires_leeway
```

Comment:
Nit:
Put spaces between the comma separated 3 items.

Reply:
I put no spaces on them to differenciate them from human comments